jquery.validate.unobtrusive.js

```diff
@@ -108,6 +108,7 @@
             .removeData("unobtrusiveContainer")
             .find(">*")  // If we were using valmsg-replace, get the underlying error
                 .removeData("unobtrusiveContainer");
+        $form.find("span").empty();
```

Comment:
You should probably be more specific than just the span. I can have spans within my form that are not related to validation.